backend/src/nodes/utils/architecture/GFPGAN/gfpgan_bilinear_arch.py

```diff
@@ -187,6 +187,7 @@ def __init__(
         self.input_is_latent = input_is_latent
         self.different_w = different_w
         self.num_style_feat = num_style_feat
+        self.min_size_restriction = 512
```

Comment:
Actually, should we assume square min sizes?

Reply:
yes, until I see something in the wild that has a rectangular minimum size I'm fine with assuming squares.